src/ui/editorwidgets/qgsrelationreferenceconfigdlgbase.ui

```diff
@@ -122,7 +122,7 @@
       <item row="5" column="0" colspan="3">
        <widget class="QCheckBox" name="mCbxChainFilters">
         <property name="toolTip">
-         <string>When activated, the filters will restrict the choices of fields to options that are </string>
+         <string>When activated, the filters will restrict the choices of fields to options that are (MISSING WORDS?)</string>
```

Comment:
Just remove?

Reply:
Maybe @m-kuhn knows (https://github.com/qgis/QGIS/commit/733b7c9c8d83062c053a13743a7a2b0afc5fce26).

Reply:
@m-kuhn ?

Reply:
https://github.com/qgis/QGIS/commit/733b7c9c8d83062c053a13743a7a2b0afc5fce26#diff-c90e64112a47b9f71d3867e4209267e6e23abbaea6b5201a6b8fbff07fb131b1R102

> Chained filters restrict the option of subsequent filters based on the selection of a previous filter.

I think we want this